client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel-mobile/incident-info-panel-mobile.component.html

```diff
@@ -106,16 +101,15 @@ <h1>Stage of Control</h1>
                 <div *ngIf="incident.fireOfNoteInd">
                     <div class="sub-content">
                         <div class="icon-title">
-                            <img class="fire-icon" style="padding-right: 18px !important" height="22" width="22" src="/assets/images/local_fire_department.png"
-                                alt="local_fire_department" />
+                            <img class="fire-icon" style="padding-right: 18px !important" height="22" width="22"
+                                src="/assets/images/local_fire_department.png" alt="local_fire_department" />
                             <span>Wildfire Of Note</span>
                         </div>
                     </div>
-                    <div class="sub-content">
+                    <div class="sub-content" style="padding-bottom: 24px;">
```

Comment:
Why not just add the style to the `sub-content` class?

Reply:
Resolved by adding new wildfire-of-note css class